### PR TITLE
add recipe for neurdflib

### PR DESCRIPTION
**Pull request by tgbugs**
per https://github.com/nipy/nipype/pull/2701
@satra

### CODE REVIEW

**Comment by satra** [comment]
@conda-forge-admin - could this be merged?

File: recipes/neurdflib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (marcelotrevisani) [inline recipes/neurdflib/meta.yaml]: Please, change this to:

```
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

File: recipes/neurdflib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (marcelotrevisani) [inline recipes/neurdflib/meta.yaml]: If this is a pure python package you can also add 
```
noarch: python
```

inside of the ``build`` section

Comment (tgbugs) [reply recipes/neurdflib/meta.yaml]: It is indeed. Thank you for the suggestion!

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to add this to ``conda-forge`` ! :)

**Pull request merged**